.circleci/config.yml

```diff
@@ -344,6 +344,38 @@ jobs:
           path: /tmp/smoketest.log
           destination: smoketest-logs
 
+  scenario-player-integration-test:
+    parameters:
+      py-version:
+        <<: *py-version-template
+
+    executor:
+      name: docker
+      py-version: << parameters.py-version >>
+
+    steps:
+      - attach-and-link:
+          py-version: << parameters.py-version >>
+      - conditional-skip
+      - config-path
+      - run:
+          name: Checkout Scenario Player
+          command: |
+            git clone https://github.com/raiden-network/scenario-player.git
+      - run:
+          name: Install dependencies
+          command: |
+            make -C scenario-player install-dev
```

Comment:
Just to make sure, the scenario player will run using the local version of raiden, and because of `config-path` that will be the version of the PR, correct?

Reply:
Correct, since `config-path` sets the venv in which raiden is already installed (in the preceding jobs) it will simply not download anything upon installation. 

You can see it in the CI run. Output here: 

`Requirement already satisfied: raiden in /home/circleci/raiden (from raiden-scenario-player==0.6.5) (0.100.5a1.dev1584+g78fe7a124)`